bindings/go/api/extern.go

```diff
@@ -20,16 +21,28 @@ func SetOnGetBlockHeaderHash(fn func(toBeHashed []byte) []byte) {
 }
 
 // SetOnGetAtv ...
-func SetOnGetAtv(fn func(id []byte) []byte) {
-	ffi.OnGetAtv = fn
+func SetOnGetAtv(fn func(idBytes entities.AtvID) []byte) {
+	ffi.OnGetAtv = func(id []byte) []byte {
+		var idBytes entities.AtvID
+		copy(idBytes[:], id)
+		return fn(idBytes)
```

Comment:
https://stackoverflow.com/questions/28886616/convert-array-to-slice-in-go

Reply:
https://stackoverflow.com/questions/30285680/how-to-convert-slice-to-fixed-size-array